src/sentry/static/sentry/app/views/projectInstall/newProject.jsx

```diff
@@ -1,46 +1,37 @@
-import React from 'react';
-import createReactClass from 'create-react-class';
 import DocumentTitle from 'react-document-title';
+import React from 'react';
 import styled from 'react-emotion';
-import space from 'app/styles/space';
-
-import OrganizationState from 'app/mixins/organizationState';
 
 import CreateProject from 'app/views/onboarding/createProject';
 import ProjectSelector from 'app/components/projectHeader/projectSelector';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import withOrganization from 'app/utils/withOrganization';
 
-const NewProject = createReactClass({
-  displayName: 'NewProject',
-  mixins: [OrganizationState],
-
-  render() {
-    const hasSentry10 = this.getFeatures().has('sentry10');
-    return (
-      <Container>
-        {!hasSentry10 && (
-          <div className="sub-header flex flex-container flex-vertically-centered">
-            <div className="p-t-1 p-b-1">
-              <ProjectSelector organization={this.getOrganization()} />
-            </div>
-          </div>
-        )}
-        <div className="container">
-          <Content>
-            <DocumentTitle title={'Sentry'} />
-            <CreateProject
-              getDocsUrl={({slug, projectSlug, platform}) => {
-                if (platform === 'other') {
-                  platform = '';
-                }
-                return `/${slug}/${projectSlug}/getting-started/${platform}`;
-              }}
-            />
-          </Content>
+const NewProject = ({organization}) => (
+  <Container>
+    {!organization.features.includes('sentry10') && (
+      <div className="sub-header flex flex-container flex-vertically-centered">
+        <div className="p-t-1 p-b-1">
```

Comment:
should we clean these up while we in here?

Reply:
This is code that will be going away soon correct? Not sure it's worth the effort. I don't know immediately what needs to be done here.